frontend/src/graphql/mutations.js

```diff
@@ -71,8 +71,18 @@ export const createUser = gql`
 `
 
 export const sendCoins = gql`
-  mutation($identifier: String!, $amount: Decimal!, $memo: String!) {
-    sendCoins(identifier: $identifier, amount: $amount, memo: $memo)
+  mutation(
+    $identifier: String!
+    $amount: Decimal!
+    $memo: String!
+    $targetCommunity: Int!
+  ) {
+    sendCoins(
+      identifier: $identifier,
+      amount: $amount,
+      memo: $memo,
+      targetCommunity: $targetCommunity
```

Comment:
i would define the targetCommunity identifier as the 1st parameter, perhaps would it rename to recipientCommunityIdentifier and would rename the 2nd parameter to recipientIdentifier

Reply:
ok